src/export.js

```diff
@@ -1,18 +1,19 @@
-// Deprecated
-// Extend assert methods to QUnit for Backwards compatibility
-( function() {
-	var i,
-		assertions = Assert.prototype;
+function applyDeprecated( name ) {
+	return function() {
+		throw new Error(
+			"The global `" + name + "` is now removed from QUnit 2.0, you should " +
+			"now use the correct namespace. Check out our upgrade guide at " +
+			"https://qunitjs.com/upgrade-guide-2.x/"
+		);
+	};
+}
 
-	function applyCurrent( current ) {
-		return function() {
-			var assert = new Assert( QUnit.config.current );
-			current.apply( assert, arguments );
-		};
-	}
+( function() {
+	var i;
+	var assertions = Assert.prototype;
```

Comment:
TL;DR: yes.

that's a mess I'm making after changes that happened on our [style guide](http://contribute.jquery.org/style-guide/js). It was a suggesting a single var and then it suggested 1 var per line, stacking non initialized variables on the same line. Now there's nothing and I'm confused.

At least on of the examples the style guide is using a single var, so I'll apply the change you suggested.